docs/api/actions-and-connectors/execute.asciidoc

```diff
@@ -50,6 +50,106 @@ depending on the connector type. For information about the parameter properties,
 refer to <<action-types>>.
 +
 --
+.Jira connectors
+[%collapsible%open]
+====
+`subAction`::
+(Required, string) The action to test. Valid values include: `fieldsByIssueType`,
+`getFields`, `getIncident`, `issue`, `issues`, `issueTypes`, and `pushToService`.
+
+`subActionParams`::
+(Required^*^, object) The set of configuration properties, which vary depending
+on the `subAction` value. This object is not required when `subAction` is
+`getFields` or `issueTypes`.
++
+.Properties of `subActionParams` when `subAction` is `fieldsByIssueType`
+[%collapsible%open]
+=====
+`id`:::
+(Required, string) The Jira issue type identifier. For example, `10024`.
+=====
++
+.Properties of `subActionParams` when `subAction` is `getIncident`
+[%collapsible%open]
+=====
+`externalId`:::
+(Required, string) The Jira issue identifier. For example, `71778`.
+=====
++
+.Properties of `subActionParams` when `subAction` is `issue`
+[%collapsible%open]
+=====
+`id`:::
+(Required, string) The Jira issue identifier. For example, `71778`.
+=====
++
+.Properties of `subActionParams` when `subAction` is `issues`
+[%collapsible%open]
+=====
+`title`:::
+(Required, string) The title of the Jira issue.
+=====
++
+.Properties of `subActionParams` when `subAction` is `pushToService`
+[%collapsible%open]
+=====
+comments:::
+(Optional, array of objects) Additional information that is sent to Jira.
++
+.Properties of `comments`
+[%collapsible%open]
+======
+comment::::
+(string) A comment related to the incident. For example, describe how to
+troubleshoot the issue.
+
+commentId::::
+(integer) A unique identifier for the comment.
+======
+
+incident:::
+(Required, object) Information necessary to create an incident in Jira.
++
+.Properties of `incident`
+[%collapsible%open]
+======
+`description`::::
+(Optional, string) The details about the incident.
+
+////
+`externalId`::::
```

Comment:
Though it's listed in https://github.com/elastic/kibana/blob/main/x-pack/plugins/actions/README.md, when I tried to use this property, I received an error:

> curl -X POST "https://localhost:5601/api/actions/connector/b3aad810-edbe-11ec-82d1-11348ecbf4a6/_execute?pretty" -H 'kbn-xsrf: true' -H 'Content-Type: application/json' -d '
> {"params":{"subActionParams":{"incident":{"summary":"Doc test 4"}, "externalId":"71778"},"subAction":"pushToService"}}' --cacert ca.crt -u elastic

>{"status":"error","message":"error validating action params: types that failed validation:\n- [0.subAction]: expected value to equal [getFields]\n- [1.subAction]: expected value to equal [getIncident]\n- [2.subAction]: expected value to equal [handshake]\n- [3.subActionParams.externalId]: definition for this key is missing\n- [4.subAction]: expected value to equal [issueTypes]\n- [5.subAction]: expected value to equal [fieldsByIssueType]\n- [6.subAction]: expected value to equal [issues]\n- [7.subAction]: expected value to equal [issue]","retry":false,"connector_id":"b3aad810-edbe-11ec-82d1-11348ecbf4a6"}

Is it a valid option?